### PR TITLE
QRadar_Article

### PR DESCRIPTION
**Pull request by David-BMS**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/42087

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by julieschwartz18** [comment]
@David-BMS - documentation reviewed. Please confirm it is ok. There are some sections that are TBD - are those supposed to stay that way or are there updates for them?

**Comment by David-BMS** [comment]
> @David-BMS - documentation reviewed. Please confirm it is ok. There are some sections that are TBD - are those supposed to stay that way or are there updates for them?

I'll remove the TBD for now, there will be some additions later on but we can merge for now

File: content-repo/extra-docs/packs/QRadar.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+id: QRadar
+title: QRadar 
+description: This pack contains all the content objects you need to interact with your QRadar SIEM, including integrations, playbooks, mappers, incident types, layouts, and scripts.
+---
+
+Use this pack's content to ingest offenses from your QRadar system and to run playbooks that perform searches and add indicators to reference sets.
+Although we recommend enabling the integration to fetch offenses from QRadar, you can use the integration even without fetching since this pack has content such as playbooks that are not related to offenses.
+
+Prior to start using the pack we recommend for you to review this detailed [article](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/ingest-incidents-from-a-siem.html) which covers how to configure the integration to fetch incident and get familiarized with working with the generic playbook and incident layout.
```
Comment (David-BMS) [inline content-repo/extra-docs/packs/QRadar.md:10]: @michalgold WDYT

File: content-repo/extra-docs/packs/QRadar.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+## Pack Workflow
+
+
+### Fetch Workflow
+Configure XSOAR to fetch incidents/offenses from QRadar to make the most of your integration.
+Configure your QRadar integration as described in the [article](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/ingest-incidents-from-a-siem.html).
```
Comment (David-BMS) [inline content-repo/extra-docs/packs/QRadar.md:17]: @michalgold And we can leave this

File: content-repo/extra-docs/packs/QRadar.md (hunk 1 above)
Comment (michalgold) [reply content-repo/extra-docs/packs/QRadar.md:10]: need to make sure it is doc-approved as well. 
Also, need to better emphasize what is in the article and what is in the doc. I think the main difference is that the documentation has more of a use case overview as well.

**Review by michalgold — APPROVED** [review]

File: content-repo/extra-docs/packs/QRadar.md (hunk 1 above)
Comment (David-BMS) [reply content-repo/extra-docs/packs/QRadar.md:10]: @julieschwartz18 Please review the last editions

**Pull request merged**